README.md

```diff
@@ -1,29 +1,3 @@
-# Google Cloud Diagnostics Common
+___This module has been renamed to [@google-cloud/common`](https://www.npmjs.com/package/@google-cloud/common). Please use that module instead.___
```

Comment:
That's not actually what happened. This was an internal module and we no longer need its functionality.

Reply:
Updated it to say "superseded". Or should I not reference the @google-cloud/common at all?